output_generators/terminal.js

```diff
@@ -17,10 +17,16 @@ function prettyPrintResult( result ){
   delete result.testCase.in.api_key; // don't display API key
 
   var input = JSON.stringify(result.testCase.in);
-  var expectationCount = result.testCase.expected.properties.length;
+  var expectationCount;
```

Comment:
Thank you for fixing this bug that I noticed yesterday. It was here before your code :)